torch_xla/experimental/custom_kernel.py

```diff
@@ -331,6 +331,24 @@ def flash_attention(
   return FlashAttention.apply(q, k, v, causal)
 
 
+def paged_attention(q, k_pages, v_pages, lengths, page_indices,
```

Comment:
how do you test this method? I see you are testing the functionality directly - via bypassing this method.

Reply:
Yep, we'll have two separate unit tests: one with directly bypassing this method and another one to test this wrapper. I updated PR with the latter.